src/doppler_env/__init__.py

```diff
@@ -1,26 +1,78 @@
 """
 Imported by doppler_env.pth Automatically inject Doppler secrets as environment variables when spawning a Python process if the `DOPPLER_ENV` environment variable is set
 """
+from base64 import b64encode
+import io
 import os
+import subprocess
+import sys
+import urllib.request
+import urllib.error
 
+from dotenv import load_dotenv
 
-def doppler_secrets_env():
-    import io
-    import subprocess
-    from dotenv import load_dotenv
 
-    DOPPLER_ENV_COMMAND = os.environ.get(
-        'DOPPLER_ENV_COMMAND', 'doppler secrets download --no-file --format env'
-    )
+def log(message):
+    print('[doppler-env]: {}'.format(message))
+
+
+def print_debug_info(doppler_token=None, project=None, config=None):
+    doppler_token and log('Token: {}'.format(doppler_token))
```

Comment:
Do we want to be redacting the token? As it stands today, the user's doppler token might end up in a log somewhere

Reply:
Good call. Done!